newsfragments/44055.feature.rst

```diff
@@ -0,0 +1 @@
+New argument ``wait_policy`` to control waiting behaviour when using ``EmrCreateJobFlowOperator``.
```

Comment:
You do not need this file, it is not a breaking change and more importantly, it is a provider change, not an Airflow change. Thus, you can just remove this file :)

Reply:
Alright. Done